cv32/bsp/README.md

```diff
@@ -46,18 +46,54 @@ user software interrupt handler must also handle exceptions.
 
 The vector table is defined in `vectors.S` and may jump to one of the
 following three interrupt request handlers in `handlers.S`:
```

Comment:
Thanks for updating the README. Very minor note - need to remove "three" now we have more than three interrupt request handlers.

Reply:
Thanks...will fix in another commit on this PR.